<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addBook</class>
 <widget class="QDialog" name="addBook">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>406</width>
    <height>130</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="lblBookName">
           <property name="text">
            <string>Book name: </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lblAthor">
           <property name="text">
            <string>Athor: </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lblRelease">
           <property name="text">
            <string>Release year: </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lblGroup">
           <property name="text">
            <string>Group: </string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLineEdit" name="lneBookName"/>
         </item>
         <item>
          <widget class="QLineEdit" name="lneAthor"/>
         </item>
         <item>
          <widget class="QLineEdit" name="lneRelease"/>
         </item>
         <item>
          <widget class="QComboBox" name="comboGroup">
           <item>
            <property name="text">
             <string>None</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Biography</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Business</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Children</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Dictionary</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>English literature</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Fiction</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>History</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Medicine</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Music</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Novel</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Persian literature</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Religious</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Sci-fic</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Science</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QPushButton" name="btnAdd">
         <property name="text">
          <string>Save</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnCancel">
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
